src/main/java/net/glowstone/GlowServer.java

```diff
@@ -2511,18 +2536,18 @@ public String getConsoleLogDateFormat() {
 
     /**
      * Gets the server type.
-     *
+     * 
      * <p>Currently, this value is set to {@code VANILLA}.
      *
      * @return the server type.
      */
     public String getServerType() {
-        return "VANILLA";
+        return strings.getString("glowstone.server-type.vanilla");
```

Comment:
I don't think this should be mutable

Reply:
It's used in packets, so yes, it should not be localized.

Reply:
Reverted.